qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/identifier/RandomPart.groovy

```diff
@@ -0,0 +1,35 @@
+package life.qbic.portal.portlet.offers.identifier
+
+/**
+ * <class short description - 1 Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since <versiontag>
+ */
```

Comment:
description missing

Reply:
```suggestion
/**
 * Represents the random part of the offer identifier.
 *
 * This class contains the value of the random part of the offer identifier.
 *
 * @since 0.1.0
 */
```